ui/config-overrides.js

```diff
@@ -16,6 +17,11 @@ module.exports = {
             environmentFilePath
         )
     );
+
+    // removes CssMinimizerPlugin() which has a bug
+    // see https://github.com/all-of-us/workbench/pull/6753#issuecomment-1150407055
+    config.optimization.minimizer = config.optimization.minimizer.filter(plugin => ! plugin instanceof CssMinimizerPlugin);
```

Comment:
Does removing this cause a significant difference in terms of CSS size (e.g. if you run yarn build --environment prod and examine the CSS outputs)? Were you able to find any reference to the underlying bug so we can revert this later once it's addressed?

Reply:
@evrii is looking into the bug - I believe it's an issue relating to trying to minimize an already-minimzed `clr`

`yarn build --environment prod` with CSS Minimizer removed
```
File sizes after gzip:

  2.76 MB (+60 B)  build/static/js/main.4f3a6ab4.js
  168.12 kB        build/static/css/main.74147a99.css
```

with the change reverted
```
  991.42 kB (-1.77 MB)  build/static/js/main.be83829e.js
  168.12 kB             build/static/css/main.74147a99.css
```

wait, that seems to be skipping the JS minimization, not the CSS!  OK this is clearly broken and I'll try again

Reply:
The filter logic was incorrect: it removed *all* filters for some reason - I don't understand untyped JS well enough to know what happened.

Switching to pop() correctly removes the second filter only.  The output difference is about 2%
```165.14 kB (-2.98 kB)```

Reply:
`minimizers` looks like this:
```
[
  {
    "options": {
      "test": {},
      "extractComments": true,
      "parallel": true,
      "minimizer": {
        "options": {
          "parse": {
            "ecma": 8
          },
          "compress": {
            "ecma": 5,
            "warnings": false,
            "comparisons": false,
            "inline": 2
          },
          "mangle": {
            "safari10": true
          },
          "keep_classnames": false,
          "keep_fnames": false,
          "output": {
            "ecma": 5,
            "comments": false,
            "ascii_only": true
          }
        }
      }
    }
  },
  {
    "options": {
      "test": {},
      "parallel": true,
      "minimizer": {
        "options": {}
      }
    }
  }
]
```

An alternative filter might be to keep the one with `"parse": { "ecma" }` perhaps

Reply:
updated to try that ^

Reply:
Looks like this is working.  Also attempted Eric's suggestion to apply CSS Minimizer in a more-focused way at #6770 but I was not successful.